### PR TITLE
Regen sql from latest swagger.

### PR DESCRIPTION
**Pull request by jaredmoo**
Swagger commit id is 723d7ee6b699121d01d2f71ccefcc7b2183e6361

### CODE REVIEW

**Comment by jaredmoo** [comment]
@TianoMS Is the markdown publish bug fixed?

**Comment by TianoMS** [comment]
@jaredmoo are you talking about the issue in publishing .md files? That issue is not fixed yet...

**Comment by TianoMS** [comment]
there are failed test cases.

**Comment by jaredmoo** [comment]
Ok, opened https://github.com/Azure/azure-resource-manager-schemas/issues/338 for the publishing issue. Fixed tests.

File: schemas/2014-04-01/Microsoft.Sql.json [hunk 1]
```diff
@@ -1539,6 +1697,52 @@
...
+      "type": "object",
+      "properties": {
+        "name": {
+          "oneOf": [
+            {
+              "type": "string"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
```
Comment (TianoMS) [inline schemas/2014-04-01/Microsoft.Sql.json]: you don't need this since "name" is a string.

Comment (jaredmoo) [inline schemas/2014-04-01/Microsoft.Sql.json]: Understood, but it's generated by AutoRest. Does it make a material difference to customer experience? If yes, we should file a bug on autorest. If no, let's leave it.

Comment (TianoMS) [inline schemas/2014-04-01/Microsoft.Sql.json]: It should not cause any behavioral difference. I'm OK with keeping it.

**Pull request merged**